src/packages/flavors/index.ts

```diff
@@ -12,9 +12,15 @@ import {
   FilecoinLegacyProviderOptions
 } from "@ganache/filecoin-options";
 
+import { TruffleColors } from "@ganache/colors";
+import chalk from "chalk";
+
 // we need "@ganache/options" in order for TS to properly infer types for `DefaultOptionsByName`
 import "@ganache/options";
 
+const NEED_HELP = "Need help? Reach out to the Truffle community at";
+const COMMUNITY_LINK = "https://gitter.im/ConsenSys/truffle";
```

Comment:
Okay, so I rustled the feathers and we now have an official support link: https://trfl.co/support ! It's not a great UX if you aren't already verified in the ConsenSys discord, but it's actively being worked on this week.

I've updated the links elsewhere in this repo here: https://github.com/trufflesuite/ganache-core/pull/874

```suggestion
const COMMUNITY_LINK = "https://trfl.co/support";
```